test/dapp/DAppSigner.test.ts

```diff
@@ -586,7 +586,7 @@ describe('DAppSigner', () => {
   })
 
   describe('signTransaction', () => {
-    it('should handle transaction without node account ids', async () => {
+    it.skip('should handle transaction without node account ids', async () => {
```

Comment:
Note, we have a separate issue to track these. https://github.com/hashgraph/hedera-wallet-connect/issues/330 

This is related to the commit we reverted recently on transaction handling.